psydac/cmd/accelerate.py

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
There is useless blank line here, which should be removed

Reply:
Done in [568885e](https://github.com/pyccel/psydac/pull/542/commits/568885e233a1430ffdec624ec8a8e90e4e30af9d).